types/meson.build

```diff
@@ -19,13 +19,17 @@ lib_wlr_types = static_library(
 		'xdg_shell_v6/wlr_xdg_shell_v6.c',
 		'xdg_shell_v6/wlr_xdg_surface_v6.c',
 		'xdg_shell_v6/wlr_xdg_toplevel_v6.c',
+		'tablet_v2/wlr_tablet_v2_pad.c',
```

Comment:
Is this alphabetical? X -> t -> w is weird, inside the subdirs it looks reasonable alphabetical.